terraform/modules/waf/main.tf

```diff
@@ -275,3 +299,15 @@ resource "aws_wafv2_regex_pattern_set" "ip_restricted_paths" {
     }
   }
 }
+
+resource "aws_wafv2_ip_set" "blocklist" {
+  provider = aws.us-east-1
+  count    = length(var.blocked_ip_addresses) > 0 ? 1 : 0
```

Comment:
Is there a problem if the list is empty? The rule assumes it will exist (directly references `aws_wafv2_ip_set.blocklist[0]`) so we should remove the count if possible.

Reply:
I've removed the count